src/Extractor/ReadAccessor.php

```diff
@@ -154,6 +154,64 @@ public function getExpression(Expr\Variable $input): Expr
         throw new CompileException('Invalid accessor for read expression');
     }
 
+    public function getIsNullExpression(Expr\Variable $input): Expr
```

Comment:
shouldn't this method be called `getIsNotNullExpression()`?

Reply:
Nope, it really give a check whether the value is null `isset($value) | null === $source->getFoo()` and not the contrary